com.regnosys.rosetta/src/com/regnosys/rosetta/generator/java/object/NamespaceHierarchyGenerator.xtend

```diff
@@ -0,0 +1,127 @@
+package com.regnosys.rosetta.generator.java.object
+
+import java.util.Collection
+import java.util.Collections
+import java.util.List
+import java.util.Map
+import java.util.concurrent.atomic.AtomicInteger
+import org.eclipse.xtext.generator.IFileSystemAccess2
+
+class NamespaceHierarchyGenerator {
+
+	val String CDM_NAMESPACE_ROOT = "cdm"
+	val String ORG_ISDA_CDM_NAMESPACE_ROOT = "org.isda.cdm"
```

Comment:
I think this is a special case that we need to cater for that is only a temp.

Maybe deprecate it and add a comment that it should be removed once namespaces are complete?

Reply:
done.